codegen/type_converter.go

```diff
@@ -71,14 +71,26 @@ type TypeConverter struct {
 	Helper        PackageNameResolver
 	uninitialized map[string]*fieldStruct
 	fieldCounter  int
+
+	requestTypeHelper ConvertOptions
+}
+
+// ConvertOptions is the helper struct for method generation
+type ConvertOptions struct {
+	FromSuffix       string
+	FromInputType    string
```

Comment:
`InputTypeName` or something.

Reply:
i think making it from/to or request/response is more clear  - 
basically it will be 
```
func OutputMethodName (in FromInputType) FromOutputType {
   out := ToType
}
```